js/admin/dnd.js

```diff
@@ -140,15 +140,29 @@ function initTableDnD(table)
 				params['selected_pagination'] = parseInt($('input[name=selected_pagination]').val());
 
 				var data = $.tableDnD.serialize().replace(/table-/g, '');
+        var updatePositionBaseUrl = typeof frameworkUpdatePositionUrl === 'string' ? frameworkUpdatePositionUrl : currentIndex + '&token=' + token;
```

Comment:
```suggestion
        var updatePositionBaseUrl = typeof frameworkUpdatePositionUrl === 'string' ? frameworkUpdatePositionUrl : currentIndex;
```

Reply:
yes, framework url has the token already. Shouldn't be added twice, ill remove one.